lib/condorLogParser.py

```diff
@@ -1310,10 +1319,10 @@ def diffTimeswWrap(start_time, end_time, year, wrap_time):
 
 
 def interpretStatus(status, default_status="Idle"):
-    """Transform an integer globus status to either Wait, Idle, Running, Held, Completed or Removed.
+    """Transforms an integer HTCondor status to either Wait, Idle, Running, Held, Completed or Removed.
```

Comment:
Should we consider quoting the different statuses in the description (something like 'Wait', 'Idle', ...)?

Reply:
no, they are not variables

Reply:
quoted

Reply:
Just confirming that this has not been quoted, which is completely okay based on the explanation above.